.github/workflows/publish-dev.yml

```diff
@@ -29,7 +29,10 @@ jobs:
         run: pnpm run cover 
 
       - name: Publish packages
+        run: |
```

Comment:
Usually the git config is done in a separate task so other tasks don't have to repeat the process if needed.

Reply:
Okay, will add a "Configure git" step before publishing